src/web/topic/components/TopicPane/TopicPane.styles.tsx

```diff
@@ -55,30 +61,25 @@ const drawerMinWidthRem = nodeWidthRem * 2 + drawerPaddingRem + drawerScrollbarW
 // So there's a parent div non-fixed position in order to allow affecting surrounding elements (e.g. menu button),
 // and this needs to match transition and size of Paper; this is why there's css on both elements.
 export const StyledDrawer = styled(Drawer, options)<DrawerProps>`
-  ${({ open, isLandscape }) => {
+  ${({ open, anchor }) => {
+    const isLandscape = anchor !== "bottom";
     const lengthIfOpen = isLandscape
       ? `${drawerMinWidthRem}rem`
       : `min(30vh, ${drawerMinWidthRem}rem)`;
     const length = open ? lengthIfOpen : "0";
-    if (isLandscape) {
-      return css`
-        width: ${length};
+    const width = isLandscape ? length : "100%";
+    const height = isLandscape ? "100%" : length;
+    const borderStyle = open ? "" : "border:none;";
+    return css`
+      width: ${width};
+      height: ${height};
 
-        & .MuiDrawer-paper {
-          width: ${length};
-        }
-      `;
-    } else {
-      return css`
-        height: ${length};
-        width: 100%;
-
-        & .MuiDrawer-paper {
-          height: ${length};
-          width: 100%;
-        }
-      `;
-    }
+      & .MuiDrawer-paper {
+        width: ${width};
+        height: ${height};
+        ${borderStyle};
```

Comment:
So when we close the drawer we arn't using display none but rather width adjustment aka `width: 0px`. The border on the drawer still takes up space (1px); this causes page overflow.

Reply:
very nice find! I feel like it would've taken me forever to figure out why the scrollbar started appearing.

This is the type of thing that would be really good to have as a comment though, since it's hard to understand otherwise. Could you add a comment to clarify? Something like this maybe
```ts
    const borderStyle = open ? "" : "border: none;"; // drawer is given a 1px border which takes up more space than the 0 width
```

Alternatively, I think setting `overflow: hidden` on the drawer would allow the 0px width to be true. Either way works

Reply:
jk i'm just gonna add this comment because it's pretty easy